### PR TITLE
Reformat args for radius installation

### PR DESCRIPTION
**Pull request by rynowak**
# Description

This change reformats the args passed into to "install radius" in the cloud functional tests. This was a single line before and really hard to review or change.

This broke in one of my recent changes.

## Type of change

- This pull request is a minor refactor, code cleanup, test improvement, or other maintenance task and doesn't change the functionality of Radius (issue link optional).

## Contributor checklist
Please verify that the PR meets the following requirements, where applicable:

- [ ] An overview of proposed schema changes is included in a linked GitHub issue.
- [ ] A design document PR is created in the [design-notes repository](https://github.com/radius-project/design-notes/), if new APIs are being introduced.
- [ ] If applicable, design document has been reviewed and approved by Radius maintainers/approvers.
- [ ] A PR for the [samples repository](https://github.com/radius-project/samples) is created, if existing samples are affected by the changes in this PR.
- [ ] A PR for the [documentation repository](https://github.com/radius-project/docs) is created, if the changes in this PR affect the documentation or any user facing updates are made.
- [ ] A PR for the [recipes repository](https://github.com/radius-project/recipes) is created, if existing recipes are affected by the changes in this PR.

### CODE REVIEW

File: .github/workflows/functional-test-cloud.yaml [hunk 1]
```diff
@@ -648,7 +648,12 @@ jobs:
           echo "*** Installing Radius to Kubernetes ***"
           rad install kubernetes \
             --chart ${{ env.RADIUS_CHART_LOCATION }} \
-            --set rp.image=${{ env.CONTAINER_REGISTRY }}/applications-rp,rp.tag=${{ env.REL_VERSION }},dynamicrp.image=${{ env.CONTAINER_REGISTRY }}/dynamic-rp,dynamicrp.tag=${{ env.REL_VERSION }},controller.image=${{ env.CONTAINER_REGISTRY }}/controller,controller.tag=${{ env.REL_VERSION }},ucp.image=${{ env.CONTAINER_REGISTRY }}/ucpd,ucp.tag=${{ env.REL_VERSION }},de.image=${{ env.DE_IMAGE }},de.tag=${{ env.DE_TAG }},global.azureWorkloadIdentity.enabled=true
+            --set rp.image=${{ env.CONTAINER_REGISTRY }}/applications-rp,rp.tag=${{ env.REL_VERSION }} \
+            --set dynamicrp.image=${{ env.CONTAINER_REGISTRY }}/dynamic-rp,dynamicrp.tag=${{ env.REL_VERSION }} \
+            --set controller.image=${{ env.CONTAINER_REGISTRY }}/controller,controller.tag=${{ env.REL_VERSION }} \
+            --set ucp.image=${{ env.CONTAINER_REGISTRY }}/ucpd,ucp.tag=${{ env.REL_VERSION }} \
+            --set de.image=${{ env.DE_IMAGE }},de.tag=${{ env.DE_TAG }} \
+            --set global.azureWorkloadIdentity.enabled=true
```
Comment (rynowak) [inline .github/workflows/functional-test-cloud.yaml:656]: Compare with: https://github.com/radius-project/radius/blob/main/.github/workflows/functional-test-noncloud.yaml#L290

**Review by ytimocin — APPROVED** [review]
What changed here other than the lines? :D

**Review by willdavsmith — APPROVED** [review]

**Comment by rynowak** [comment]
> What changed here other than the lines? :D

Just the lines.

**Review by kachawla — APPROVED** [review]

**Comment by rynowak** [comment]
As discussed, I'll hold off on this change until after the release. I'm in no rush to break the build again.

**Comment by rynowak** [comment]
Hey everyone, I'm going to merge this and hope that it works. I'll babysit the build in case this breaks something.

**Pull request merged**